### DISCUSSION TITLE
1.48 (Error: Pool send a seed hash for an odd dagEpoch (-1) ...)

### DISCUSSION THREAD

**Discussion by ryanyeap** [Lolliedieb/lolMiner-releases#1469] [Q&A]
Anyone know what does this mean?
![Untitled](https://user-images.githubusercontent.com/2985313/161410358-3dcf4839-0c32-4f0a-8c57-455c947c87b7.jpg)

Win11 / 512.15

![Untitled2](https://user-images.githubusercontent.com/2985313/161410543-9b219433-3fe0-4c69-90b9-4c0b6e2ae007.jpg)

**Comment by egantz09** [comment]
I have the same problem, did anyone solved it?

**Reply by STDevl** [reply]
This is a user error, he is trying to mine Ethereum Classic on the Ethereum pool.

**Reply by egantz09** [reply]
Thanks, I corrected this, thank you again for the info.

**Comment by STDevl** [comment]
ETChash is the algorithm for mining ETC, not for ETH.
